setup.cfg

```diff
@@ -38,6 +38,7 @@ install_requires =
 
 [options.extras_require]
 tests =
```

Comment:
@phanikumv i think we should have a section for `all` which should contain open lineage and another section called openlineage like in https://github.com/astronomer/astronomer-providers/blob/main/setup.cfg#L118